apps/src/code-studio/header.js

```diff
@@ -240,9 +240,9 @@ function remixProject() {
     // page or a script level. In these cases, copy will create a new project
     // for us.
     var newName = "Remix: " + (dashboard.project.getCurrentName() || appOptions.level.projectTemplateLevelName || "My Project");
-    dashboard.project.copy(newName, function () {
+    dashboard.project.copy(newName, {shouldNavigate: true}).then(() => {
       $(".project_name").text(newName);
-    }, {shouldNavigate: true});
+    });
```

Comment:
You don't handle a possible promise rejection right now.  I prefer to do that at the outermost layer if possible, since error handling usually involves some kind of global (even if just calling `console.error`, which helps users report issues to us) - so this seems like a good place for a `.catch()` case.